### PR TITLE
feat(editor): Add support for `unplugin-icons` and auto-importing components (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

- Adds support for `unplugin-icons`
  - Icons are auto-imported, just use them like `<IconMdiAccountBox />`
- Adds support for `unplugin-vue-components`

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/CAT-10/setup-unplugin-icons-in-editor-ui

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/vite.config.mts [hunk 1]
```diff
@@ -74,6 +76,14 @@ const plugins = [
 		compiler: 'vue3',
 		autoInstall: true,
 	}),
+	components({
```
Comment (mutdmour) [inline packages/editor-ui/vite.config.mts:79]: could we add this to the design system? so we could use it there while also still being able to use it directly here

for example, how would we use this with `n8n-icon` or `n8n-button`?

Comment (alexgrozav) [reply packages/editor-ui/vite.config.mts:79]: Added to the design system. I don't have the time to replace the FontAwesome usages with this, so we'll need to leave that for another time.

**Review by mutdmour — DISMISSED** [review]

**Review by mutdmour — DISMISSED** [review]

**Review by mutdmour — DISMISSED** [review]

**Review by mutdmour — APPROVED** [review]
😻

**Comment by janober** [comment]
Got released with [`n8n@1.52.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.52.0)

**Pull request merged**